lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,42 +2,58 @@ module ActiveModel
   class Serializer
     module Adapter
       class Attributes < Base
+        attr_accessor :_nested_associations
+
         def serializable_hash(options = nil)
           options ||= {}
+
+          self._nested_associations = ActiveModel::Serializer::Utils
```

Comment:
I personally think it could make sense to override it here in certain cases, but for now let's remain consistent and work on it in a subsequent PR.